Discord link for World MoveIt Day (#448)
ros-planning/moveit.ros.org: _posts/2020-04-29-world-moveit-day-2020.md

Join the live stream on [this Google Meet](https://meet.google.com/_meet/cjk-qoym-qzz), which supports up to 250 simultaneous users.

> Join the conversation on **Discord** (different for *Discourse*, similar to Slack) in the new [MoveIt Server](https://discord.gg/RrySut8). Register instantly [here](https://discord.gg/RrySut8).

Reviewer: ```suggestion
Join the conversation on **Discord** (a chat client, different from *Discourse*) in the new [MoveIt Server](https://discord.gg/RrySut8). Register instantly [here](https://discord.gg/RrySut8).
```
